### PR TITLE
New UI for the Event Details Landing Page - PizzaDAO

### PR DESCRIPTION
**Pull request by NikhilMahana**
# Description

Enhancing the current Event Details Landing Page to have a better user-experience.

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [x] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

## Release Note Draft Snippet
As requested, here's the New UI for the Event Details Page.

Attaching screenshot for reference:

image

Thanks for the opportunity to contribute to Unlock, @julien51! Looking forward to your feedback!

Feel free to connect with me on Twitter: [@Nikhil_Mahana](http://x.com/Nikhil_Mahana/)

### CODE REVIEW

**Comment by julien51** [comment]
Can you please make sure the toolbar is displayed below the image?

**Comment by NikhilMahana** [comment]
> Can you please make sure the toolbar is displayed below the image?

You mean this? The suggestion from Snax?

image

**Comment by julien51** [comment]
yes that is what I meant. Also please make sure the tests pass, as well as make sure you post a screenshot of how it looks like on mobile. Our UI is currently responsive. We need to make sure this is still the case.

File: unlock-app/src/components/content/EventContent.tsx [hunk 1]
```diff
@@ -1,81 +1,426 @@
...
+  event: Event
   checkoutConfig: {
     id?: string
     config: PaywallConfigType
   }
 }
 
-export const EventContentWithProps = ({
-  isLoading,
+const language = () => {
```
Comment (julien51) [inline unlock-app/src/components/content/EventContent.tsx:51]: I am not sure I understand why this is here?

**Review by julien51 — COMMENTED** [review]
Looking at the code I am very confused about why you added a bunch of code that was in another component?

**Comment by NikhilMahana** [comment]
> yes that is what I meant. Also please make sure the tests pass, as well as make sure you post a screenshot of how it looks like on mobile. Our UI is currently responsive. We need to make sure this is still the case.

Gotcha, will make sure about the responsiveness as well as try to document it better.

**Comment by NikhilMahana** [comment]
> Looking at the code I am very confused about why you added a bunch of code that was in another component?

Let me work with the previous version of code to optimise it and minimise the risk of test failure. I'll let you know with the developments.

**Comment by julien51** [comment]
Closing in favor of https://github.com/unlock-protocol/unlock/pull/13869

**Pull request closed without merging**